src/triage/cli.py

```diff
@@ -436,5 +438,37 @@ def history(self, args):
         db_history(dburl=self.root.db_url)
 
 
+    @cmdmethod('password', action='store_true', help='hidden password prompt')
+    @local
+    def up(self, args):
+        (retcode, stdout, stderr) = self.local['docker']['container', 'inspect', '-f', "'{{.State.Status}}'", 'triage_db'].run(retcode=None)
+        if retcode != 0:
+            logger.info('Container does not exist')
+            if os.path.exists('database.yaml'):
```

Comment:
This is checking in the working directory where the `triage` command is being run from, right? That's probably fine, but should make sure the documentation calls out that the CLI cares what directory it's being run from